src/main/java/hlf/java/rest/client/controller/FabricClientController.java

```diff
@@ -224,4 +225,32 @@ public ResponseEntity<EventAPIResponseModel> getTransactionByBlockNumber(
     return transactionFulfillment.readTransactionEventByBlockNumber(
         blockNumber, networkName, transactionId, chaincode, eventType);
   }
+
+  /**
+   * The REST Endpoint for supporting write transactions to ledger by accepting a Payload that can
+   * consists of multiple Private Data Collection details as well as multiple Public data.
+   *
+   * @param channelName String channel name
+   * @param chaincodeName String chaincode name
+   * @param functionName String function name in chaincode
+   * @param payload a Request body of kind {@link MultiDataTransactionPayload}
+   * @return responseEntity ResponseEntity Transaction Response
+   */
+  @PostMapping(value = "/invoke_transaction/multi_data")
```

Comment:
Will the endpoint `/invoke_transaction` be used for PDC after this change?

Reply:
It can be used only exception being only single private/public data can be passed. A much more cleaner approach would be a single write endpoint that can cater to single or multiple public/private data unlike how invoke_transaction would be behaving today. Probably those changes would change the current behaviour of existing APIs and probably should to target that for a major release. Introduced a new API currently since we didnt was to disrupt any of the existing functionalities.

Reply:
I agree, we can have a single endpoint for all the invoke calls

Reply:
[Part 1]
[nitpick] How about following the versions for the API in such a case?

For example, the URI can be `/invoke_transaction/v2` for the newly added code. The current API `/invoke_transaction` can be marked for deprecation. This will let the swagger and/or Java doc reflect to the users that the current API is being planned for deprecation.

[Part 2]
Apart from the API behavior, the same versioning scheme can be applied to the Kafka payload. Carrying both formats would be tedious in later releases. Let's mark the behaviors for deprecation as time permits.

Reply:
Sure, lets apply versioning. 
As a separate task, we can also re-work or deprecate other APIs as part of a major release.